src/in-depth/signals-ctrlc.rs

```diff
@@ -1,7 +1,9 @@
+use std::{thread, time::Duration};
+
 fn main() {
     ctrlc::set_handler(move || {
         println!("received Ctrl+C!");
-    }).expect("Error setting Ctrl-C handler");
-
-    // ...
```

Comment:
The `// ...`  is short hand for _here your actual code that can be "crtl-c-ed"_.